easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20230930-R2-intel-compilers-2022.1.0.eb

```diff
@@ -0,0 +1,49 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2017 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+#
+# Authors::
+# * Kenneth Hoste <kenneth.hoste@ugent.be>
+# * Ward Poelmans <wpoely86@gmail.com>
+# * Oliver Stueker <oliver.stueker@ace-net.ca>
+# License::   MIT/GPL
+##
+name = 'GAMESS-US'
+version = '20230930-R2'
+
+ddi_comm = 'sockets'
+maxcpus = '1024'
+maxnodes = '1'
+hyperthreading = False
+
+homepage = 'https://www.msg.chem.iastate.edu/gamess/'
+description = """The General Atomic and Molecular Electronic Structure System (GAMESS)
+is a general ab initio quantum chemistry package.
+This module can be used on a maximum of %s nodes and %s CPUs per node.
+ """ % (maxnodes, maxcpus)
+
+toolchain = {'name': 'intel-compilers', 'version': '2022.1.0'}
+toolchainopts = {'openmp': True}
+
+dependencies = [
+    ('imkl', '2022.1.0', '', SYSTEM),
+]
+
+download_instructions = """
+1. Manually download via http://www.msg.chem.iastate.edu/gamess/download.html (requires registration)
+2. Rename gamess-current.tar.gz to gamess-%(version)s.tar.gz
+"""
+sources = ['gamess-%(version)s.tar.gz']
+patches = ['GAMESS-US-20230930_add_slurm_support_mpi_target.patch']
```

Comment:
As that is a generic patch, how about removing the version of it? Right now the file name in the EasyConfig file and the patch in this PR having different file names. That string of numbers confuses me all the time.

Reply:
This patch does not apply to versions of GAMESS-US prior to 2023xxxx, and we don't know if it will apply to future versions. So the version tag in the filename is very fitting.